### DISCUSSION TITLE
Hospital-based Home Health Agency

### DISCUSSION THREAD

**Discussion by Ftokic** [CMSgov/hospital-price-transparency#50] [Q&A] [answered]
Hospital-based Home Health Agency - Does this service line need to be included in the MRF hospital file?

**Comment by RidenourK** [comment, marked as answer]
@Ftokic. Thank you for your question. 

The machine-readable file must include all items and services for which the hospital has established a standard charge. For purposes of complying with the hospital price transparency requirements that are effective as of January 1, 2021, items and services are all items and services, including individual items and services and service packages, that could be provided by a hospital to a patient in connection with an inpatient admission or an outpatient department visit for which a hospital has established a standard charge. Examples include supplies and procedures, room and board, and use of the facility and other items (generally described as facility fees), services of employed physicians and non-physician practitioners (generally reflected as professional charges), and any other item or service for which a hospital has established a standard charge. Please refer to [45 CFR §180.20](https://www.ecfr.gov/current/title-45/section-180.20).
